common/JackAPI.cpp

```diff
@@ -510,7 +510,7 @@ LIB_EXPORT int jack_port_tie(jack_port_t* src, jack_port_t* dst)
         jack_error("jack_port_tie called with ports not belonging to the same client");
         return -1;
     } else {
-        return manager->GetPort(mydst)->Tie(mysrc);
+        return (manager ? manager->GetPort(mydst)->Tie(mysrc) : -1);
```

Comment:
I think the check should be on the `if` above.
this should do it right?
```
if (manager || manager->GetPort(mysrc)->GetRefNum() != manager->GetPort(mydst)->GetRefNum()) {
```

Reply:
Meant to say `manager == nullptr` there

Reply:
Well, it will prevent a possible crash, but it'll produce a misleading error message.
The other functions in the code use the same kind of in-place check, which is why I used that here.

Reply:
right, I see that now. a bit weird because it returns a generic -1 error without any info or warning log, but ok let's keep the code consistent.
will merge, thanks!